tools/expotools/src/commands/PublishPackages.ts

```diff
@@ -934,6 +934,14 @@ async function _updatePodsAsync(allConfigs: Map<string, PipelineConfig>): Promis
       cwd: path.join(EXPO_DIR, 'ios'),
     });
   }
+  if (await _promptAsync(`Do you want to update pods in ${chalk.magenta('expo/apps/bare-expo/ios')}?`)) {
+    // Update all pods that have been published
+    console.log(`\nUpdating pods: ${chalk.green(podspecNames.join(' '))}`);
+
+    await _spawnAsync('pod', ['update', ...podspecNames, '--no-repo-update'], {
```

Comment:
Hmm, this array of podspec names may not be valid for bare-expo, it contains pods included in the client - afaik we exclude some packages in bare-expo so I'd expect this command can fail in some cases. Imho better to just wait for publish packages v2 that handles this 🙂

Reply:
How about running `pod install` in the directory? That's what I was doing right now and it should make no unnecessary changes if the `Podfile` isn't invalid.

Currently my workflow for publishing packages is:
1. `et pp …`
2. enter enter enter
3. run `pod install` in `bare-expo/ios`
4. amend the commit
5. (if I remember) update changelog, amend the commit again
6. push

if a couple of lines would make it easier to publish packages I would be for it

Reply:
yeah, `pod install` would work as a temporary solution, however we used `update` intentionally - it's much faster.

Reply:
…or not
```
> time pod update EXNotifications --no-repo-update
pod update EXNotifications --no-repo-update  9,73s user 7,72s system 78% cpu 22,344 total
pod update EXNotifications --no-repo-update  8,71s user 4,58s system 102% cpu 12,987 total
pod update EXNotifications --no-repo-update  8,62s user 4,46s system 101% cpu 12,833 total
```
```
> time pod install
pod install  8,70s user 4,42s system 102% cpu 12,802 total
pod install  7,20s user 3,65s system 103% cpu 10,534 total
pod install  4,74s user 2,38s system 102% cpu 6,959 total
```
```
> time pod update EXNotifications --no-repo-update
pod update EXNotifications --no-repo-update  4,72s user 2,40s system 102% cpu 6,955 total
pod update EXNotifications --no-repo-update  4,69s user 2,39s system 102% cpu 6,929 total
pod update EXNotifications --no-repo-update  4,60s user 2,32s system 102% cpu 6,742 total
```
```
> time pod install
pod install  4,78s user 2,43s system 102% cpu 7,053 total
pod install  4,54s user 2,30s system 102% cpu 6,661 total
pod install  4,60s user 2,30s system 102% cpu 6,720 total
```

Reply:
After some consideration (after pushing latest commit, introducing more elaborate logic around `pod update` and `pod install`) I think I no longer believe we should ask if the user would like to `pod install` if `pod update` failed.

Maybe after a failed `pod update` and a reconfirmation from the user that they wanted to update the pod we should abort the publish process (since there must be something wrong if the pod is supposed to be updated in `bare-expo` but it couldn't)?

Reply:
I think just running `pod install` (without trying to update) is good enough, but it's up to you.